test/Reflection/ReflectionMethodTest.php

```diff
@@ -121,7 +121,7 @@ public function testMethodDocBlockFromParent()
         $prototypes = $laminasReflectionMethod->getPrototypes();
         list($first, $second) = $prototypes[1]->getParameters();
 
-        self::assertEquals('\\' . Node::class, $first->getType());
+        self::assertEquals('self', $first->getType());
```

Comment:
This might be hinting at a potential issue if code using reflection is not aware of those special typehints. Please look at the usage and expectations, they might need to be transparently resolved to actual class name.